packages/js/passkeys-sdk/README.md

```diff
@@ -0,0 +1,54 @@
+# Hanko Passkeys JavaScript SDK
+
+[View docs on docs.hanko.io](https://docs.hanko.io/passkey-api/js-sdk)
```

Comment:
This should also be removed until the pages are available.

Reply:
They will be available as soon as this PR goes through